### DISCUSSION TITLE
Strimzi.io certificate expired

### DISCUSSION THREAD

**Discussion by kacperurb** [strimzi/strimzi-kafka-operator#9491] [General]
Hi Team,
as you can see on the screenshot below certificate expired.
It was valid untill "Wednesday, December 27, 2023 at 2:54:32 PM"
Can you do something about it?
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/27071337/1494b0fe-75db-43ca-a9ca-b662fce980df)

Regards,
Kacper

**Comment by scholzj** [comment]
There seemed to be some issue in Netlify (or between Netlify and Let's Encrypt), but it seems to be fixed now and the certificate is renewed as far as I can see.

**Reply by kacperurb** [reply]
Thanks @scholzj, it looks good now. 
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/27071337/09194457-aa80-40d9-9361-0f25f2fe2fe5)
